.github/workflows/run-tests.yml

```diff
@@ -16,6 +16,8 @@ jobs:
           # brotlipy. Moving pypy3 to only test linux.
           - python-version: pypy3
             os: ubuntu-latest
+          - python-version: 3.10.0-beta.4
```

Comment:
Would setting this to `3.10-dev` mean we get release candidates?

Reply:
That's a great question! I'd just pulled this out of the urllib3 without much research, let me take a look.

Reply:
`3.10-dev` it is! Thanks for the callout, I just moved it into the main runner list for all platforms since it's already passing. 🎉